mxcube3/routes/Utils.py

```diff
@@ -227,4 +232,44 @@ def take_snapshots(self, snapshots=None, _do_take_snapshot=_do_take_snapshot):
 def enable_snapshots(collect_object):
     # patch collect object
     collect_object.take_crystal_snapshots = types.MethodType(take_snapshots, collect_object)
-     
+
+
+def send_mail(_from, to, subject, content):
+    smtp = smtplib.SMTP('smtp', smtplib.SMTP_PORT)
+    date = email.Utils.formatdate(localtime=True)
+
+    email_msg = "From: %s\r\nTo: %s\r\nSubject: %s\r\nDate: %s\r\n\r\n%s" % (
+        _from, to, subject, date, content)
+
+    try:
+        error_dict = smtp.sendmail(_from, to.split(','), email_msg)
+    except smtplib.SMTPException, e:
```

Comment:
can we add the content dat to the log output in case of exception? so it does not get lost

Reply:
I've, added some more information to the logs, you cant see it here though because your comment is on the row above the affected lines